apps/demos/Demos/Stepper/FormIntegration/description.md

```diff
@@ -0,0 +1,6 @@
+This demo shows the integration of DevExtreme Stepper, [Form](/Documentation/Guide/UI_Components/Form/Overview/), and [MultiView](/Documentation/Guide/UI_Components/MultiView/Overview/), including validation for both Form and Stepper.
```

Comment:
"Integration" by itself is too generic. I'm not that familiar with all available DevExtreme widgets, so I had to review the code to understand what goes where. Where's the Multi View, where's the Form, and what their purpose is. So I think we should add a few additional details to this introduction. 

```suggestion
This demo uses DevExtreme Stepper to implement navigation in for hotel registration UI. Input controls on each step are organized with the help of [MultiView](/Documentation/Guide/UI_Components/MultiView/Overview/) and [Form](/Documentation/Guide/UI_Components/Form/Overview/) components. Form and Stepper display input validation results when a user navigates to the next step.
```